tests/integration/test_interface.py

```diff
@@ -221,7 +221,7 @@ def test_column_noising(dataset_name: str, user_config, request):
             # and additional parameters to consider as well as the rtol when the
             # number of compared is small.
             expected_noise = 1 - (1 - CELL_PROBABILITY) ** len(col.noise_types)
-            rtol = 0.5 if includes_token_noising else 0.11
+            rtol = 0.55 if includes_token_noising else 0.11
```

Comment:
Same issue caused here by middle initial here.  And I figured we could leave improving this for the existing ticket to do so.